src/test/ui/cenum_impl_drop_cast.rs

```diff
@@ -0,0 +1,18 @@
+#![deny(cenum_impl_drop_cast)]
+
+enum E {
+    A = 0,
+}
+
+impl Drop for E {
+    fn drop(&mut self) {
+        println!("Drop");
+    }
+}
+
+fn main() {
+    let e = E::A;
+    let i = e as u32;
+    //~^ ERROR Cast `enum` implementing `Drop` `E` to integer `u32`
```

Comment:
oh, shoot, this will have to be adjusted to match the new error =)

Reply:
That's a small fix :-)